.github/workflows/ios_tests.yml

```diff
@@ -15,7 +15,17 @@ jobs:
       - uses: actions/checkout@v1
         with:
           submodules: true
-      - name: 'Install dependencies'
+      - id: check_context
+        name: 'Check whether to run'
+        run: |
+          if git rev-parse --abbrev-ref HEAD | grep -q ^main$ || git diff --name-only origin/main | grep -qe common/ -e objective-c/ -e swift/ ; then
```

Comment:
Would this also catch upstream envoy updates?

Reply:
Good point, let me add that. Will also add WORKSPACE and .bazelrc.